subt/control_center_qt.py

```diff
@@ -197,7 +197,13 @@ def run(self):
             except osgar.bus.BusShutdownException:
                 return
             if channel == "robot_status":
-                robot_id, (x, y, heading), status = data
+                robot_id, pose_or_artf, status = data
+                if len(pose_or_artf) == 3:
+                    x, y, heading = pose_or_artf
+                else:
+                    artf, x, y, z = pose_or_artf
+                    heading = 0
+                    print(pose_or_artf)
```

Comment:
This is ugly hack. The reported artifact should come into the control center on its own channel.